### DISCUSSION TITLE
Another suggestion to fashion developers (1.7.10). Pantsir missile system

### DISCUSSION THREAD

**Discussion by letovskiiy** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1366] [Q&A]
Greetings, dear developers! 
To be even more interested in the gameplay with the HBM's mod, I would very much like you to create a "Pantsir-S1" air defense system. To make this installation present characteristics as in Wikipedia, including the appearance of the air defense system: KAMAZ, etc. I literally fell in love with this car! 

You can watch a video about this system, how it works, so that the sound of launching missiles is similar, and also I would like, if you still start developing, to draw all the details of this machine as realistically as possible.

You can call this air defense system differently, but the characteristics and appearance of the system wanted to be as in the photo, as from Wikipedia.

Thank you for your work! 
Wiki: https://en.m.wikipedia.org/wiki/Pantsir_missile_system
![IMG_6294](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/160428452/3875f5f1-286c-4354-905d-8f386dd6f8a5)
![image](https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/assets/160428452/6507c48f-8e54-43c7-9ed8-ace8b5995405)

**Comment by JameH2** [comment]
I dont think vehicles are planned

Also no suggestions pls thx

**Reply by letovskiiy** [reply]
It's a pity that they are not planned, I would like to 😓

**Reply by gmhs022** [reply]
Vehicles would be wayy to hard to add, even trains are barely complete despite loads of work done on them, vehicles would be a whole other beast. The only way they could be added would be to outright ignore all other aspects of the mod and only focus on vehicles for months, possibly over a year, on end.
So yeah, prolly not being added.

**Reply by letovskiiy** [reply]
Thank you for your work anyway! Well, tell me a very good idea, cool, although very difficult, I will be waiting for new trains, but I hope that you will implement and pantsir!! I believe in you!!!

**Comment by letovskiiy** [comment]
By the way, guys, you don't have to make this car on wheels!

**Comment by JameH2** [comment]
Thats really great but i doubt the developer has some specific missile system in mind

**Comment by asddsa3211233** [comment]
Чел если хочешь хаймарс без знака всу могу скинуть файл модели
